doc/getstarted/build_and_install/docker_install_en.rst

```diff
@@ -8,199 +8,255 @@ Please be aware that you will need to change `Dockers settings
+      nvidia-docker run -it --rm paddlepaddle/paddle:0.10.0rc2-gpu /bin/bash
 
-    Please be aware that the CPU-only and the GPU images both use the AVX
-    instruction set, but old computers produced before 2008 do not support
-    AVX.  The following command checks if your Linux computer supports
-    AVX:
+2. development image :code:`paddlepaddle/paddle:<version>-dev`
 
-    .. code-block:: bash
+   This image has packed related develop tools and runtime
+   environment. Users and developers can use this image instead of
+   their own local computer to accomplish development, build,
+   releasing, document writing etc. While different version of paddle
+   may depends on different version of libraries and tools, if you
+   want to setup a local environment, you must pay attention to the
+   versions.  The development image contains:
+   
+   - gcc/clang
+   - nvcc
+   - Python
+   - sphinx
+   - woboq
+   - sshd
+     
+   Many developers use servers with GPUs, they can use ssh to login to
+   the server and run :code:`docker exec` to enter the docker
+   container and start their work.  Also they can start a development
+   docker image with SSHD service, so they can login to the container
+   and start work.
 
-       if cat /proc/cpuinfo | grep -i avx; then echo Yes; else echo No; fi
 
+Train Model Using Python API
+----------------------------
 
-       If it doesn't, we will use the non-AVX images.
+Our official docker image provides a runtime for PaddlePaddle
+programs. The typical workflow will be as follows:
 
-    Above methods work with the GPU image too -- just please don't forget
-    to install GPU driver. To support GPU driver, we recommend to use 
-    [nvidia-docker](https://github.com/NVIDIA/nvidia-docker). Run using
+Create a directory as workspace:
 
-    .. code-block:: bash
+.. code-block:: bash
 
-        nvidia-docker run -it --rm paddledev/paddle:0.10.0rc1-gpu /bin/bash
+   mkdir ~/workspace
 
-    Note: If you would have a problem running nvidia-docker, you may try the old method we have used (not recommended).
+Edit a PaddlePaddle python program using your favourite editor
 
-    .. code-block:: bash
+.. code-block:: bash
 
-        export CUDA_SO="$(\ls /usr/lib64/libcuda* | xargs -I{} echo '-v {}:{}') $(\ls /usr/lib64/libnvidia* | xargs -I{} echo '-v {}:{}')"
-        export DEVICES=$(\ls /dev/nvidia* | xargs -I{} echo '--device {}:{}')
-        docker run ${CUDA_SO} ${DEVICES} -it paddledev/paddle:<version>-gpu
+   emacs ~/workspace/example.py
 
+Run the program using docker:
 
-3. Use production image to release you AI application
-    Suppose that we have a simple application program in :code:`a.py`, we can test and run it using the production image:
+.. code-block:: bash
 
-    ```bash
-    docker run -it -v $PWD:/work paddle /work/a.py
-    ```
+   docker run -it --rm -v ~/workspace:/workspace paddlepaddle/paddle:0.10.0rc2 python /workspace/example.py
```

Comment:
No need to use `-it` when you want to run a python program once. Same as below GPU version.